tooling/package.json

```diff
@@ -1,9 +1,9 @@
 {
   "private": true,
   "scripts": {
-    "lint": "jscs"
+    "lint": "cd .. && jscs"
```

Comment:
Well. This is not possible, as we append additional parameters in ```run.sh```. This would result in such calls:

    pushd .. && jscs && popd "examples/aurelia" "tests/"

;-) But this might work:

    "scripts": {
      "lint": "pushd .. && jscs",
      "postlint": "popd
    }